packages/date-picker/src/DatePicker/components/Calendar/CalendarClasses.ts

```diff
@@ -1,30 +1,27 @@
+/* eslint-disable camelcase */
+import { ClassNames } from "react-day-picker"
 import calendarStyles from "./Calendar.scss"
-
-export const defaultCalendarClasses = {
-  container: "DayPicker",
-  wrapper: `${calendarStyles.wrapper} DayPicker-wrapper`,
-  interactionDisabled: `${calendarStyles.interactionDisabled} DayPicker--interactionDisabled`,
-  months: `${calendarStyles.months} DayPicker-Months`,
-  month: `${calendarStyles.month} DayPicker-Month`,
-  navBar: "DayPicker-NavBar",
-  navButtonPrev: "DayPicker-NavButton DayPicker-NavButton--prev",
-  navButtonNext: "DayPicker-NavButton DayPicker-NavButton--next",
-  navButtonInteractionDisabled: "DayPicker-NavButton--interactionDisabled",
-  caption: `${calendarStyles.caption} DayPicker-Caption`,
-  weekdays: "DayPicker-Weekdays",
-  weekdaysRow: "DayPicker-WeekdaysRow",
-  weekday: `${calendarStyles.weekday} DayPicker-Weekday`,
-  body: `${calendarStyles.body} DayPicker-Body`,
-  week: "DayPicker-Week",
-  weekNumber: "DayPicker-WeekNumber",
-  day: `${calendarStyles.day} DayPicker-Day`,
-  footer: "DayPicker-Footer",
-  todayButton: "DayPicker-TodayButton",
-  today: `${calendarStyles.today} DayPicker-Day--today`,
-  selected: `${calendarStyles.selected} DayPicker-Day--selected`,
-  disabled: `${calendarStyles.disabled} DayPicker-Day--disabled`,
-  outside: `${calendarStyles.outside} DayPicker-Day--outside`,
-  from: `${calendarStyles.from} DayPicker-Day--from`,
-  to: `${calendarStyles.to} DayPicker-Day--to`,
-  range: `${calendarStyles.range} DayPicker-Day--range`,
+export const defaultCalendarClasses: ClassNames = {
+  root: `${calendarStyles.container}`,
```

Comment:
```suggestion
  root: calendarStyles.container,
```

This can be applied to all of them :)